zoobc/zoobc-core
# change initial score for node admittance to 20% of max score

## Description
Based on the meeting 14 January 2021, @bzpython requested the initial score that a node get upon admitted to node registry to 20% (from 10% previously).

## Breakdown
- [x] Update the initial score to 20%

## Reference Issue
Close #numberIssue

## Step to Test (optional)
- Start a node
- Register a node to the blockchain
- Observe that upon admitted the node gets 20% of max score in the beginning
